apps/web-client/src/routes/orders/suppliers/reconcile/[id]/+page.svelte

```diff
@@ -68,9 +68,11 @@
 		}
 	});
 
+	$: processedOrderDelivery = processOrderDelivery(data?.reconciliationOrderLines, data?.placedOrderLines);
```

Comment:
Would it make sense to optimize it when the user is on the first screen, and only calculate this if we're past the first step?

```suggestion
	$: processedOrderDelivery = currentStep > 1 ? processOrderDelivery(data?.reconciliationOrderLines, data?.placedOrderLines) : [];
```

Reply:
Premature optimisation is the root of all evil...
I've accepted it and it introduced a rabbit hole-inducing regression...I back pedalled for the purpose of this PR...if this really starts causing problems, I think it should be tackled as separate unit of work